### PR TITLE
PAE-412 Fix file upload using celery

### PR DESCRIPTION
**Pull request by luismorenolopera**
Description
---------------
This fixes an issue with uploading files that exceed the limit of SCORMXBLOCK_ASYNC_THRESHOLD. For some large files the connection to S3 was lost and it did not reconnect.

### CODE REVIEW

**Review by diegomillan — APPROVED** [review]
LGTM

File: scormxblock/scormxblock.py [hunk 1]
```diff
@@ -156,7 +156,7 @@ def updoad_all_content(temp_directory, fs):
         s3_upload(all_content, temp_directory, dest_dir)
     else:
         # The raw number of files is going to make this request time out. Use celery instead
-        s3_upload.apply_async((all_content, temp_directory, dest_dir), serializer='pickle')
+        s3_upload.delay(all_content, temp_directory, dest_dir)
```
Comment (Squirrel18) [inline scormxblock/scormxblock.py:159]: is it necessary to change apply_async for delay? @luismorenolopera

Comment (luismorenolopera) [reply scormxblock/scormxblock.py:159]: no.  but now as no extra arguments are used it seems correct to use delay

Comment (Squirrel18) [reply scormxblock/scormxblock.py:159]: Ok.

**Review by Squirrel18 — APPROVED** [review]

**Pull request merged**